resources/assets/js/features/search/components/Result/_Modals/SubscribeModal.js

```diff
@@ -0,0 +1,268 @@
+                                            </div>
+                                            <div className="filter__prop">
+                                                <div className="filter__prop-control">
+                                                    <div className="filter__prop-name subscribe-modal-name">
+                                                        {translate('search_result.free_seats')}
+                                                    </div>
+                                                    <select name="seats" value={seats || ''} className="form-control" id="seats" onChange={this.seatsChange}>
+                                                        <option value="">{translate('search_result.not_important')}</option>
+                                                        <option value="1">1</option>
+                                                        <option value="2">2</option>
+                                                        <option value="3">3</option>
+                                                        <option value="4">{translate('search_result.more')}4</option>
+                                                    </select>
+                                                </div>
+                                            </div>
+                                            <div className="filter__prop">
+                                                <div className="filter__prop-control">
+                                                    <div className="filter__prop-name subscribe-modal-name">
+                                                        {translate('search_result.driver_rating')}
+                                                    </div>
+                                                    <select name="rating" value={rating || ''} className="form-control" id="rating" onChange={this.ratingChange}>
+                                                        <option value="">{translate('search_result.not_important')}</option>
+                                                        <option value="1">1</option>
+                                                        <option value="2">2</option>
+                                                        <option value="3">3</option>
+                                                        <option value="4">4</option>
+                                                        <option value="5">5</option>
+                                                    </select>
+                                                </div>
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                        )
+                    }
+                    </ModalBody>
+                    {(requestSendSuccess) ? ('') : (
```

Comment:
specify method that will return needed html base on state, no need to render template parts in this way